features/step_definitions/save_occupations_steps.rb

```diff
@@ -12,7 +12,7 @@
 end
 
 Then(/^I should see a confirmation that the occupation is saved$/) do
-  confirmation_message = %("#{specific_search_result.fetch(:title)}" saved)
+  confirmation_message = %(#{specific_search_result.fetch(:title)} saved)
```

Comment:
May as well flip this back to using double quotes. I shouldn't have done something ~~clever~~ confusing in the first place